methods.tex

```diff
@@ -263,11 +263,13 @@ \subsubsection{Supply and Demand}
 for some set of commodities. Accordingly, for each request, suppliers of that
 commodity denote production capacities and an isotopic profile of the commodity
 they can provide. Suppliers are allowed to offer the null set of isotopics as
-their profile, effectively providing no information. Suppliers are also allowed
+their profile, effectively providing no information. 
```

Comment:
the previous sentence here is more about the fact that requests can be physics/chemical neutral. I'll try to clean up language here to make sure that's more clear.